chains/evm/.solhint-test.json

```diff
@@ -17,6 +17,7 @@
     "no-unused-import": "error",
     "gas-struct-packing": "warn",
     "interface-starts-with-i": "warn",
+    "import-path-check": "off",
```

Comment:
New rule in 5.1, but it doesn't get forge imports somehow. Disabled for tests only